spanner/api/Spanner.Samples.Tests/SpannerFixture.cs

```diff
@@ -102,68 +103,45 @@ public async Task InitializeAsync()
         }
     }
 
-    public Task DisposeAsync()
+    public async Task DisposeAsync()
```

Comment:
This will leave around older databases that would previously have been cleaned up, e.g. if a test was aborted abruptly. Is that okay? Do we have a separate cleanup program?

Reply:
I deleted everything that we had by hand. We don't have a cleanup process but I can set up one if we need to.
The rationaly before was that we kept the DBs around in case we needed to check for failures, in reality we never did that and it was causing quota issues. I'm hoping the case of test abruptly crashing is rare enough that I can just delete by hand again if we need to.